src/word-utils.js

```diff
@@ -31,12 +31,24 @@ export function expandOneWordToAcronym(CapitalWord) {
  * @example World Health Organization -> WHO
  */
 export function expandWordsToAcronym(words) {
-    if (words.length === 1) {
+    if (words.length == 1) {
         return expandOneWordToAcronym(words[0]);
     }
+    else{
     // World Health Organization -> WHO
-    return words.reduce((acronym, word) => {
-        acronym += word.charAt(0);
-        return acronym;
-    }, "");
+        let result = [];
+
+        let output = words.reverse().reduce((acronym, word, i) => {
```

Comment:
Do you use `output` in this code?

I believe that we can change `words.reverse().reduce` * 2 to `wrods.reverse().forEach`

```
const result = [];
const acronym = [];
const acronyms = [];

words.reverse().reduce((acronyms, word, i) => {
  // 
});
```

Reply:
For example, given `var words = ["In", "American", "Broadcast", "Company"];`,
the `output` is `["C", "BC", "ABC", "IABC"]`, the `output_1` is `["I", "IA", "IAB", "IABC"]`. In this way we can avoid capitalized prepositions at the front and back.

Reply:
No, I don't think I need `output` either.